src/components/crown/crownConfig/crownConfig.vue

```diff
@@ -102,7 +103,11 @@ export default {
   },
   mixins: [poolLaneCrownConfig],
   watch: {
-    highlightedShapes() {
+    highlightedShapes(shapes, prevShapes) {
```

Comment:
For how this works, see https://vuejs.org/v2/api/#vm-watch. There is one catch:
> Note: when mutating (rather than replacing) an Object or an Array, the old value will be the same as new value because they reference the same Object/Array. Vue doesn’t keep a copy of the pre-mutate value.

Since we create a new array each time `highlightedShapes` changes, this works as expected.